azurelinuxagent/ga/exthandlers.py

```diff
@@ -76,11 +76,14 @@
 
 
 class ExtCommandEnvVariable(object):
-    DisableFailed = "AZURE_GUEST_AGENT_DISABLE_FAILED"
-    UninstallFailed = "AZURE_GUEST_AGENT_UNINSTALL_FAILED"
-    ExtensionPath = "AZURE_GUEST_AGENT_EXTENSION_PATH"
-    ExtensionVersion = "AZURE_GUEST_AGENT_EXTENSION_VERSION"
+    Prefix = "AZURE_GUEST_AGENT"
+    DisableReturnCode = "%s_DISABLE_CMD_EXIT_CODE" % Prefix
+    UninstallReturnCode = "%s_UNINSTALL_CMD_EXIT_CODE" % Prefix
+    ExtensionPath = "%s_EXTENSION_PATH" % Prefix
+    ExtensionVersion = "%s_EXTENSION_VERSION" % Prefix
     ExtensionSeqNumber = "ConfigSequenceNumber"  # At par with Windows Guest Agent
+    NotRun = "NotRun"
```

Comment:
Is this the default value? Can you add a comment here if it is?

Reply:
It kind of is, whenever we call enable without running uninstall first, we set the env value to this.  It is to reduce ambiguity for the extension publisher where they dont have to guess if the current enable call is during an extension update case or just a normal call. Will add a comment

Reply:
If it is a value rather than a variable name probably it should not be in ExtCommandEnvVariable

Reply:
>  It is to reduce ambiguity for the extension publisher where they dont have to guess if the current enable call is during an extension update case or just a normal call

do explain :)

Reply:
>If it is a value rather than a variable name probably it should not be in ExtCommandEnvVariable
I created the `ExtCommandEnvVariable` class to contain all the environment variables in one place for better readability. I think since the `NotRun` value is set as an environment variable, makes sense to keep it there. 

 >do explain :)
It'll be easier to explain this offline. We can stay behind after scrum and I can explain the reasoning behind this